pages/01.basics/05.questions-and-answers/default.v4.md

```diff
@@ -36,7 +36,8 @@ Here you can either find the answer directly or find help to point you in the ri
 | | |
 | - | - | - |
 | **4** | **Q:** | **Where is the QLC+ user folder located in my system?** |
-| | **A:** | The user folder is where user fixtures, input profiles, RGB scripts and MIDI templates go.<br>It changes depending on your operating system:<br>**Linux**: it is a hidden folder in your user home directory: `$HOME/.qlcplus`<br>**Windows**: it is a folder in your user (e.g. MyUser) directory: `C:\\Users\\MyUser\\QLC+`<br>**Mac OS**: it is located in your user `Library` directory: `$HOME/Library/Application\\ Support/QLC+`<br>You can reach any of these folders from a terminal with the `cd` command. For example:<br><br>`cd $HOME/Library/Application\\ Support/QLC+`<br><br> Please keep in mind that fixures and input profiles found in the user folder will have precedence over the same files in the QLC+ system folder. |
+| | **A:** | The user folder is where user fixtures, input profiles, RGB scripts and MIDI templates go.<br>It changes depending on your operating system:<br>**Linux**: it is a hidden folder in your user home directory: `$HOME/.qlcplus`<br>**Windows**: it is a folder in your user (e.g. MyUser) directory: `C:\\Users\\MyUser\\QLC+`<br>**Mac OS**: it is located in your user `Library` directory: `$HOME/Library/Application\\ Support/QLC+`<br>You can reach any of these folders from a terminal with the `cd` command. For example:<br><br>`cd $HOME/Library/Application\\ Support/QLC+`<br><br> Please keep in mind that fixures and input profiles found in the user folder will have precedence over the same files in the QLC+ system folder. <br>
```

Comment:
Adding a newline in the middle will break the table.
Please remove

Reply:
Fixed